lib/config-migrator.js

```diff
@@ -0,0 +1,59 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default async function checkConfigAndMigrate () {
+  latex.log.group('LaTeX Check')
```

Comment:
Let's not log anything unless there's an error? If everything works as intended, and we have not migrated any configuration, we should be silent IMHO. The stuff we are doing here is of no relevance to users unless we actually had to migrate something. I think it's questionable even in that scenario whether we need to inform them.